ladybug/src/main/java/nl/nn/ibistesttool/Servlet.java

```diff
@@ -44,7 +44,7 @@ public ApplicationInstance newApplicationInstance() {
 
 	@Override
 	public String getUrlMapping() {
-		return "/testtool";
+		return "/iaf/testtool";
```

Comment:
Ik denk niet dat het handig is de oude gui url aan te passen.

Reply:
Waarom niet?

Reply:
Redirect de oude naar de nieuwe? Dan werken de oude en nieuwe allebei, maar kunnen we de oude wel op een gegeven moment opruimen.

Reply:
Nu wel. :)